### ISSUE TITLE
HttpConnector: controlling verbosity of default log message for incoming request

### ISSUE THREAD

**Issue by cunger** [reshufflehq/reshuffle#481]
It's a small thing: The HttpConnector currently logs `'Handling event'` for each incoming request:

https://github.com/reshufflehq/reshuffle/blob/59caf291537c884b9359810548b01cfc6739506f/src/connectors/HttpConnector.ts#L53

I'm not sure what your strategy is, but would it be possible to either remove this log message or add a way to control verbosity in this case? 

The reason I'm asking is that my Reshuffle app is running in the cloud and the load balancer pings it every minute or so to see whether it's healthy. I want the health endpoint to be silent, because it just adds a lot of superfluous logging that hides the more important log messages.

**Comment by cunger** [comment]
Ah, I guess that's what the HEALTH_CHECK_PATH environment variable is for? 😬

**Comment by duckranger** [comment]
Yeah @cunger - that's exactly what it's for. But I think your point still stands. We will look into this. Thanks!
